frame/ethereum/src/tests/mod.rs

```diff
@@ -32,6 +32,7 @@ use fp_self_contained::CheckedExtrinsic;
 mod eip1559;
 mod eip2930;
 mod legacy;
+mod merge;
```

Comment:
looks like trie tests did not run with the CI
```suggestion
mod merge;
mod trie;
```